src/tcgwars/logic/impl/gen3/FireRedLeafGreen.groovy

```diff
@@ -449,6 +449,7 @@ public enum FireRedLeafGreen implements LogicCardInfo {
             text "Once during your turn (before your attack), you may choose any 1 card from your deck and put it into your hand. Shuffle your deck afterward. You can’t use more than 1 Quick Search Poké-Power each turn. This power can’t be used if Pidgeot is affected by a Special Condition."
             actionA {
               checkNoSPC()
+              assert !(self.specialConditions) : "Pidgeot is affected by a Special Condition."
```

Comment:
This one is completely unnecessary due to the line above.

Reply:
Yep removing all those now